students/KannanSethu/session2/FizzBuzz.py

```diff
@@ -0,0 +1,13 @@
+def fizzbuzz():
+	'''function to execute fizzbuzz'''
+    for i in range(1,101):
```

Comment:
Ideally, you wouldn't hard code the range in here so the function could be more flexible.  For example, in this case you could say:
`def fizzbuzz(n):`
`---->for i in range(1, n):`
Then, you could pass any number into your function.